src/expr/src/scalar/func.rs

```diff
@@ -1360,6 +1361,18 @@ fn get_byte<'a>(a: Datum<'a>, b: Datum<'a>) -> Result<Datum<'a>, EvalError> {
     Ok(Datum::from(i32::from(*i)))
 }
 
+pub fn constant_time_compare_bytes(a: &[u8], b: &[u8]) -> Result<Datum<'a>, EvalError> {
```

Comment:
Looks like you may have forgotten to update the arguments to be `Datum` and the signature to have a generic lifetime `'a` to this function?

Reply:
Not only that, I see I also have an error in 1366 (let b = a). Thanks for the catch.